src/suricata.c

```diff
@@ -2678,14 +2678,18 @@ int InitGlobal(void) {
 
     (void)SCSetThreadName("Suricata-Main");
 
-    /* Ignore SIGUSR2 as early as possble. We redeclare interest
+    /* Set subsystem name - TLS for main thread */
+    SCSetSubsystem("Suricata-main");
```

Comment:
Is the "sub-system" name now being set 2x for every thread?  As a thread name and a sub-system name? Can they be one and the same?

Reply:
Good point. Some of the changes are pretty old. I'll rework this to integrate the thread name with the subsystem name.